app/models/micropost.rb

```diff
@@ -15,8 +15,10 @@ class Micropost < ApplicationRecord
                     size: {less_than: Settings.micropost.image_size,
                            message: I18n.t("microposts.image_size")}
 
-  scope :order_created_at ->{order(created_at: :desc)}
-  scope :micropost_feed, ->id {where("user_id = ?", id)}
+  scope :order_created_at, ->{order created_at: :desc}
+  scope :micropost_feed, (lambda do |following_ids, id|
+    where("user_id IN ? OR user_id = ?", following_ids, id)
```

Comment:
```suggestion
    where(user_id: user_ids)
```
truyền vào 1 mảng user_ids đi e

Reply:
vâng